profiles/k8s-container/tuned.conf

```diff
@@ -0,0 +1,57 @@
+# tuned configuration
+#
+
+[main]
+summary=Optimize for CPU partitioning
```

Comment:
Here should be some good summary what the profile is good for. It's for users to orientate in the profiles.